page.css

```diff
@@ -0,0 +1,98 @@
+
+html,
+body {
+    background-color: #FFE1E9;
+    font-family: sans-serif;
+}
+
+h1 {
+    font-size: 60px;
+}
+
+@media only screen and (min-width: 375px) {
+
+    .body__image {
+        display: flex;
+        height: 150px;
+
+    }
+    
+    .page_container {
+        display: block;
+        background-color: white;
+    }
+    
+    
+    .body_maincard {
+        background-color: white;
+    }
+    
+    .body__top_layout {
+         background-color: white;
+         padding: 50px;
+    
+    }
+    
+    p {
+        font-size: 33px; 
+    }
+
+
+    h1 {
+    display: block;
+    }
+
+    textarea {
+        border-radius: 15px;
+        width: 99%;
```

Comment:
99? looks strange

Reply:
For some reason, I got 1% more space on the right side of 'textarea' block. Tring to find a way to fix it